nilearn/plotting/img_plotting.py

```diff
@@ -447,12 +447,12 @@ def plot_anat(anat_img=MNI152TEMPLATE, cut_coords=None,
             you wish to save figures with a black background, you
             will need to pass "facecolor='k', edgecolor='k'"
             to matplotlib.pyplot.savefig.
-        dim : boolean or float, optional
-            Dimming factor applied to background image. If True, automatic
+        dim : float, optional
```

Comment:
> I am focussing on user specified inputs here. 'auto' is by default
> anyway doesn't require to say user to give 'auto' again. If you think
> its necessary I can add it.

Yes, it's necessary to add it and to describe what it does. Elsewhere the
user cannot interpret what he/she reads in the function signature, and
hence the default behavior.